scripts/git.py

```diff
@@ -14,5 +12,5 @@ def tags():
     """
     command = ["git", "tag", "--sort=committerdate"]
     result = run(command, capture_output=True, check=True)
-    tags = (tag.strip() for tag in result.stdout.decode("utf-8").splitlines())
-    return list(tags)
+    tags_generator = (tag.strip() for tag in result.stdout.decode("utf-8").splitlines())
```

Comment:
stupid question, why don't you use a list comprehension instead of an iterator

Reply:
Good question I think I may have returned a generator in the original impl. and did not adjust the code well enough when it grew.